2025/03/27 16:21:50 [error] 32#32: *2 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: , request: "GET / HTTP/1.1", upstream: "http://172.18.0.3:3000/", host: "localhost"
2025/03/27 16:21:51 [error] 32#32: *2 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: , request: "GET /favicon.ico HTTP/1.1", upstream: "http://172.18.0.3:3000/favicon.ico", host: "localhost", referrer: "http://localhost/"
2025/03/27 16:45:12 [error] 29#29: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: , request: "GET / HTTP/1.1", upstream: "http://172.18.0.3:3000/", host: "localhost"
2025/03/27 16:45:12 [error] 29#29: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: , request: "GET /favicon.ico HTTP/1.1", upstream: "http://172.18.0.3:3000/favicon.ico", host: "localhost", referrer: "http://localhost/"
2025/03/27 16:50:30 [error] 29#29: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: , request: "GET / HTTP/1.1", upstream: "http://172.18.0.6:3000/", host: "localhost"
2025/03/27 16:50:30 [error] 29#29: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: , request: "GET /favicon.ico HTTP/1.1", upstream: "http://172.18.0.6:3000/favicon.ico", host: "localhost", referrer: "http://localhost/"
2025/03/27 17:06:46 [error] 29#29: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: , request: "GET / HTTP/1.1", upstream: "http://172.18.0.6:3000/", host: "localhost", referrer: "http://localhost/"
2025/03/27 17:06:46 [error] 29#29: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: , request: "GET /favicon.ico HTTP/1.1", upstream: "http://172.18.0.6:3000/favicon.ico", host: "localhost", referrer: "http://localhost/"
2025/04/07 04:49:02 [error] 30#30: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: , request: "GET / HTTP/1.1", upstream: "http://172.18.0.6:3000/", host: "localhost", referrer: "http://localhost/"
2025/04/07 04:49:02 [error] 30#30: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: , request: "GET /favicon.ico HTTP/1.1", upstream: "http://172.18.0.6:3000/favicon.ico", host: "localhost", referrer: "http://localhost/"
2025/04/20 03:46:03 [error] 29#29: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: , request: "GET / HTTP/1.1", upstream: "http://172.18.0.3:3000/", host: "localhost"
2025/04/20 05:05:18 [error] 33#33: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: , request: "GET /writing_area HTTP/1.1", upstream: "http://172.18.0.3:3000/writing_area", host: "localhost"
2025/04/27 20:31:18 [error] 33#33: *2 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: , request: "GET /writing_area HTTP/1.1", upstream: "http://172.18.0.3:3000/writing_area", host: "localhost", referrer: "http://localhost/"
2025/05/01 16:19:12 [emerg] 1#1: cannot load certificate "/etc/letsencrypt/live/icit.lazyinwork.com/fullchain.pem": BIO_new_file() failed (SSL: error:80000002:system library::No such file or directory:calling fopen(/etc/letsencrypt/live/icit.lazyinwork.com/fullchain.pem, r) error:10000080:BIO routines::no such file)
2025/05/01 16:19:15 [emerg] 1#1: cannot load certificate "/etc/letsencrypt/live/icit.lazyinwork.com/fullchain.pem": BIO_new_file() failed (SSL: error:80000002:system library::No such file or directory:calling fopen(/etc/letsencrypt/live/icit.lazyinwork.com/fullchain.pem, r) error:10000080:BIO routines::no such file)
2025/05/01 16:19:17 [emerg] 1#1: cannot load certificate "/etc/letsencrypt/live/icit.lazyinwork.com/fullchain.pem": BIO_new_file() failed (SSL: error:80000002:system library::No such file or directory:calling fopen(/etc/letsencrypt/live/icit.lazyinwork.com/fullchain.pem, r) error:10000080:BIO routines::no such file)
2025/05/01 16:19:18 [emerg] 1#1: cannot load certificate "/etc/letsencrypt/live/icit.lazyinwork.com/fullchain.pem": BIO_new_file() failed (SSL: error:80000002:system library::No such file or directory:calling fopen(/etc/letsencrypt/live/icit.lazyinwork.com/fullchain.pem, r) error:10000080:BIO routines::no such file)
2025/05/01 16:19:19 [emerg] 1#1: cannot load certificate "/etc/letsencrypt/live/icit.lazyinwork.com/fullchain.pem": BIO_new_file() failed (SSL: error:80000002:system library::No such file or directory:calling fopen(/etc/letsencrypt/live/icit.lazyinwork.com/fullchain.pem, r) error:10000080:BIO routines::no such file)
2025/05/01 16:19:22 [emerg] 1#1: cannot load certificate "/etc/letsencrypt/live/icit.lazyinwork.com/fullchain.pem": BIO_new_file() failed (SSL: error:80000002:system library::No such file or directory:calling fopen(/etc/letsencrypt/live/icit.lazyinwork.com/fullchain.pem, r) error:10000080:BIO routines::no such file)
2025/05/01 16:19:25 [emerg] 1#1: cannot load certificate "/etc/letsencrypt/live/icit.lazyinwork.com/fullchain.pem": BIO_new_file() failed (SSL: error:80000002:system library::No such file or directory:calling fopen(/etc/letsencrypt/live/icit.lazyinwork.com/fullchain.pem, r) error:10000080:BIO routines::no such file)
2025/05/01 16:19:32 [emerg] 1#1: host not found in upstream "backend" in /etc/nginx/nginx.conf:116
2025/05/01 16:19:53 [emerg] 1#1: host not found in upstream "backend" in /etc/nginx/nginx.conf:116
2025/05/01 16:20:23 [emerg] 1#1: cannot load certificate "/etc/letsencrypt/live/icit.lazyinwork.com/fullchain.pem": BIO_new_file() failed (SSL: error:80000002:system library::No such file or directory:calling fopen(/etc/letsencrypt/live/icit.lazyinwork.com/fullchain.pem, r) error:10000080:BIO routines::no such file)
2025/05/01 16:21:15 [emerg] 1#1: host not found in upstream "backend" in /etc/nginx/nginx.conf:116
2025/05/01 16:21:26 [emerg] 1#1: host not found in upstream "backend" in /etc/nginx/nginx.conf:116
2025/05/01 16:21:35 [emerg] 1#1: host not found in upstream "backend" in /etc/nginx/nginx.conf:116
2025/05/01 16:21:40 [emerg] 1#1: host not found in upstream "backend" in /etc/nginx/nginx.conf:116
2025/05/01 16:21:45 [emerg] 1#1: host not found in upstream "backend" in /etc/nginx/nginx.conf:116
2025/05/01 16:21:55 [emerg] 1#1: host not found in upstream "backend" in /etc/nginx/nginx.conf:116
2025/05/06 19:44:08 [error] 29#29: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: , request: "GET /About_ HTTP/1.1", upstream: "http://172.18.0.6:3000/About_", host: "localhost", referrer: "http://localhost/About_"
2025/05/06 19:44:08 [error] 29#29: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: , request: "GET /favicon.ico HTTP/1.1", upstream: "http://172.18.0.6:3000/favicon.ico", host: "localhost", referrer: "http://localhost/About_"
2025/05/15 07:35:06 [error] 29#29: *2 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: , request: "GET /about HTTP/1.1", upstream: "http://172.18.0.7:3000/about", host: "localhost"
